### PR TITLE
Generalize default cases to terminal cases

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

**Comment by mernst** [comment]
Temporarily closing

**Comment by mernst** [comment]
@smillst This is ready for review.

File: docs/manual/creating-a-checker.tex [hunk 1]
```diff
@@ -2036,7 +2036,7 @@
 Typical use is:
 
 \begin{Verbatim}
-javac -processor mypackage.MyProcessor -Aflowdotdir=. MyClass.java
+javacheck -processor mypackage.MyProcessor -Aflowdotdir=. MyClass.java
```
Comment (smillst) [inline docs/manual/creating-a-checker.tex:2039]: The manual uses `javac` instead of `javacheck` everywhere else.

**Review by smillst — DISMISSED** [review]
One minor comment.

**Pull request merged**